metric_learn/lmnn.py

```diff
@@ -185,7 +185,7 @@ def _select_targets(self):
     target_neighbors = np.empty((self.X_.shape[0], self.k), dtype=int)
     for label in self.labels_:
       inds, = np.nonzero(self.label_inds_ == label)
-      dd = pairwise_distances(self.X_[inds])
+      dd = euclidean_distances(self.X_[inds], self.X_[inds], squared=True)
```

Comment:
You can leave out the second argument, similar to how `pairwise_distances` works.